core/src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,99 @@
+    <Loggers>
+        <!-- LOGGERS FOR UNDERLYING PACKAGES -->
+        <Logger name="org.hibernate" level="${env:RAPHTORY_SQL_LOG:-off}">
+        </Logger>
+        <Logger name="com.scurrilous.circe.checksum.Crc32cIntChecksum" level="${env:RAPHTORY_CHECKSUM_LOG:-off}">
+        </Logger>
+        <Logger name="io.netty" level="${env:RAPHTORY_NETTY_LOG:-off}">
+        </Logger>
+        <Logger name="io.grpc.netty" level="${env:RAPHTORY_NETTY_LOG:-off}">
+        </Logger>
+        <Logger name="org.apache" level="${env:RAPHTORY_APACHE_LOG:-off}">
+        </Logger>
+        <Logger name="com.typesafe.akka" level="${env:RAPHTORY_AKKA_LOG:-WARN}">
+        </Logger>
+        <Logger name="akka" level="${env:RAPHTORY_AKKA_LOG:-ERROR}">
+        </Logger>
+        <Logger name="org.asynchttpclient" level="${env:RAPHTORY_APACHE_LOG:-off}">
+        </Logger>
+
+        <!-- LOGGER FOR RAPHTORY CORE -->
+        <Logger name="com.raphtory" level="${defaultLogLevel}">
+        </Logger>
+
+        <!-- LOGGERS FOR RAPHTORY PACKAGES -->
+        <Logger name="com.raphtory.algorithms" level="${env:RAPHTORY_ALGORITHMS_LOG:-${defaultLogLevel}}">
+        </Logger>
+        <Logger name="com.raphtory.api" level="${env:RAPHTORY_API_LOG:-${defaultLogLevel}}">
+        </Logger>
+       <Logger name="com.raphtory.formats" level="${env:RAPHTORY_FORMATS_LOG:-${defaultLogLevel}}">
+        </Logger>
+        <Logger name="com.raphtory.sinks" level="${env:RAPHTORY_SINKS_LOG:-${defaultLogLevel}}">
+        </Logger>
+        <Logger name="com.raphtory.spouts" level="${env:RAPHTORY_SPOUTS_LOG:-${defaultLogLevel}}">
+        </Logger>
+        <Logger name="com.raphtory.utils" level="${env:RAPHTORY_UTIL_LOG:-${defaultLogLevel}}">
+        </Logger>
+
+        <!-- LOGGERS FOR RAPHTORY INTERNALS PACKAGES -->
+        <Logger name="com.raphtory.internals" level="${defaultInternalsLogLevel}">
+        </Logger>
+        <Logger name="com.raphtory.internals.communication" level="${env:RAPHTORY_INTERNALS_COMMUNICATION_LOG:-${defaultInternalsLogLevel}}">
+        </Logger>
+        <Logger name="com.raphtory.internals.graph" level="${env:RAPHTORY_INTERNALS_GRAPH_LOG:-${defaultInternalsLogLevel}}">
+        </Logger>
+        <Logger name="com.raphtory.internals.management" level="${env:RAPHTORY_INTERNALS_MANAGEMENT_LOG:-${defaultInternalsLogLevel}}">
+        </Logger>
+        <Logger name="com.raphtory.internals.serialisers" level="${env:RAPHTORY_INTERNALS_SERIALISERS_LOG:-${defaultInternalsLogLevel}}">
+        </Logger>
+        <Logger name="com.raphtory.internals.storage" level="${env:RAPHTORY_INTERNALS_STORAGE_LOG:-${defaultInternalsLogLevel}}">
+        </Logger>
+        <Logger name="com.raphtory.internals.time" level="${env:RAPHTORY_INTERNALS_TIME_LOG:-${defaultInternalsLogLevel}}">
+        </Logger>
+
+
+        <!-- LOGGERS FOR RAPHTORY COMPONENTS -->
+        <Logger name="com.raphtory.internals.components" level="${defaultInternalsComponentsLogLevel}">
```

Comment:
Are all these package paths correct still, I think some of them are old and can be removed